Wip - few updates

Quentinchampenois opened the pull request:
#### :tophat: What? Why?

Few updates on UI

#### :pushpin: Related Issues
- Related to https://github.com/armandfardeau/decidim/pull/12#issuecomment-620143349

#### :clipboard: Subtasks
- [x] Swap the alert css class for "Examinated" and "Classified"
- [x] Display the answer date below the title
- [x] Allows admin to edit signature date collection for "examinated", "debatted" and "examinated" initiative state
- [x] Add tests
- [x] Backport commit fixing github actions

Quentinchampenois committed:
swap Examinated & Classified css class

Quentinchampenois committed:
refactor architecture and 'answer_date' format

Quentinchampenois committed:
display 'signature_date' in edit form for examinated initiative state

Quentinchampenois committed:
update initiative helper specs

mrcasals committed:
Update sassc gem version

(cherry picked from commit 04c157d3554e7a38fa30df14656295ba6e0d6511)

Quentinchampenois committed:
signature date collection editable for several states

Quentinchampenois committed:
update tests and rubocop

Quentinchampenois committed:
update workflows for running on all branches

Quentinchampenois committed:
udpate workflows to run on wip branches

Quentinchampenois committed:
i18n normalization

Quentinchampenois committed:
update initiative spec

Quentinchampenois committed:
rollback to initial github actions

Quentinchampenois committed:
remove changes on github actions

armandfardeau commented on decidim-initiatives/app/forms/decidim/initiatives/admin/initiative_answer_form.rb:
```diff
@@ -42,6 +42,17 @@ def answer_date_allowed?
+
+        def check_state
+          case context.initiative.state
+          when "published", "debatted", "examinated", "classified"
```
I may be better to Use the constant defined on Decidim::Initiatives class, then add published?

Quentinchampenois replied:
Hi, thanks for your suggestion, I update it

armandfardeau commented on decidim-initiatives/spec/system/initiative_spec.rb:
```diff
@@ -55,7 +55,7 @@
 
       it "displays date" do
-        expect(page).to have_content(I18n.l(base_initiative.answer_date, format: :short))
+        expect(page).to have_content(I18n.l(base_initiative.answer_date, format: "%d/%m/%Y"))
```
Please use I18n format

Quentinchampenois replied:
hi, thanks for the review. I agree with you and update it

armandfardeau reviewed (changes requested):
Thanks for your contribution, may you correct a few things before merging?

Quentinchampenois committed:
Replace date format with i18n format

Because 'answer_date' is a not a date but a time, I have to cast it to date to match the i18n :decidim_short format

Quentinchampenois committed:
refactor initative state checking
